ant-protocol/src/storage/scratchpad.rs

```diff
@@ -57,43 +106,32 @@ impl Scratchpad {
         self.data_encoding
     }
 
-    /// Increments the counter value.
-    pub fn increment(&mut self) -> u64 {
+    /// Updates the content, re-signs the scratchpad
```

Comment:
better expand `Updates the content` to more detailed work done:
* increment the counter
* encrypt the input content and replace the existing encrypted data
* sign the new fields and replace the existing signature

Reply:
Agreed!